### ISSUE TITLE
[MNG-7438] add execution id to "Configuring mojo xxx with basic configurator" debug message

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#9289]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** opened **[MNG-7438](https://issues.apache.org/jira/browse/MNG-7438?redirect=false)** and commented

currently, when running goals/mojos, we have in debug logs

```
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ org.example.apache.maven.execution ---
[DEBUG] Configuring mojo org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test from plugin realm ClassRealm[plugin>org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5, parent: sun.misc.Launcher$AppClassLoader@7852e922]
[DEBUG] Configuring mojo 'org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test' with basic configurator -->
[DEBUG]   (s) additionalClasspathElements = []
[DEBUG]   (s) basedir = /home/herve/tmp/org.example.apache.maven.execution
```

on the "`Configuring mojo 'org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test' with basic configurator`" line, it completely hides the fact that configuration is valid for an execution id, that has a default value when nobody defined any (see https://maven.apache.org/guides/mini/guide-default-execution-ids.html )

Just adding "`execution 'default-test'`" would make things more clear so people learn about execution ids

```
[DEBUG] Configuring mojo 'org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test' execution 'default-test' with basic configurator -->
```

## final debug messages

```
[DEBUG] Loading mojo org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test from plugin realm ClassRealm[plugin>org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5, parent: sun.misc.Launcher$AppClassLoader@7852e922]
[DEBUG] Configuring mojo execution 'org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test:default-test' with basic configurator -->
```

---

**Affects:** 3.8.5

**Remote Links:**
- [GitHub Pull Request #700
](https://github.com/apache/maven/pull/700)

**Backported to:** [4.0.0-alpha-2](https://github.com/apache/maven/milestone/113?closed=1), [3.9.0](https://github.com/apache/maven/milestone/2?closed=1), [3.8.8](https://github.com/apache/maven/milestone/109?closed=1)

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

fixed in https://github.com/apache/maven/commit/3c6009cbf48f78bc8f657c1eb18fa8f522a994a5 for 3.9.x
and https://github.com/apache/maven/commit/643d1e782800e52bc2b8a85aa949e1c62b8f980a for 4.0.0-alpha

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [64da2c58243bc6f48827301b744b9f6312497550](https://gitbox.apache.org/repos/asf?p=maven.git&a=commit&h=64da2c58243bc6f48827301b744b9f6312497550) for `maven-3.8.x` branch.
